tests/integration/relationships/belongs-to-test.js

```diff
@@ -211,8 +211,8 @@ module('integration/relationship/belongs_to Belongs-To Relationships', {
     });
 
     Comment = Message.extend({
-      body: DS.attr('string'),
-      message: DS.belongsTo('message', { polymorphic: true, async: false, inverse: null }),
+      body: DSattr('string'),
+      message: DSbelongsTo('message', { polymorphic: true, async: false, inverse: null }),
```

Comment:
shouldn't this be `DS.attr` and `DS.belongsTo`?

Reply:
Agree.

Reply:
They're aliased at the top of the file - I've copied the style of the other tests

Reply:
You're right. Sorry I didn't see that.